Bynder/Test/Api/RequestSender/ApiRequestSenderTest.cs

```diff
@@ -15,23 +15,28 @@ namespace Bynder.Test.Api.RequestSender
 {
     public class ApiRequestSenderTest
     {
-        [Fact]
-        public async Task WhenRequestIsPostThenParametersAreAddedToContent()
+        private const string ACCES_TOKEN = "access_token";
```

Comment:
This might not be anything big but maybe we call it `ACCESS_TOKEN` instead just to stay consistent?